build.sbt

```diff
@@ -52,7 +30,6 @@ lazy val root = (project in file(".")).aggregate(
     fapiClient_play27,
     fapiClient_play28
   ).settings(
-    publishArtifact := false,
     publish / skip := true,
```

Comment:
`publishArtifact := false` is unnecessary when `publish / skip := true` is set.

`publish / skip` is the recommended key for this purpose, and is the key examined by `sbt-version-policy` when it checks to see if it can skip calculating compatibility for a particular subproject.